README.rst

```diff
@@ -110,6 +110,32 @@ need to be specified as it is computed from the supplied ``pixel_order``.
     pixel = neopixel.NeoPixel(board.D0, 1, pixel_order=neopixel.RGBW)
     pixel[0] = (30, 0, 20, 10)
 
+Setup for sudo-less usage on Raspberry Pi boards
+=============
+1. Enable both SPI and Serial port hardware (Serial interface). Do it by ``raspi-config`` tool or manually by adding
+
+   ::
+   
```

Comment:
```suggestion

```

Reply:
Can you do it as a `.. code-block::` and then omit the spaces? I don't have any more CI info. My next step would be to run pre-commit locally to see what it changes. (Some other libraries do print out a diff I think. NeoPixel may not have it.)